packages/calcite-components/src/components/radio-button/radio-button.tsx

```diff
@@ -286,11 +289,25 @@ export class RadioButton
     });
   }
 
+  private updateTabIndexOfOtherRadioButtonsInGroup(): void {
+    const radioButtons = this.queryButtons();
+    const otherFocusableRadioButtons = radioButtons.filter(
+      (radioButton) => radioButton.guid !== this.guid && !radioButton.disabled
+    );
+    otherFocusableRadioButtons.forEach((radiobutton) => {
+      forceUpdate(radiobutton);
+    });
+  }
```

Comment:
I tested completely removing `updateTabIndexOfOtherRadioButtonsInGroup` and this works great with just the changes you made to `isFocusable`.  I tested with multiple radio button groups and tabbing between groups works correctly.  Try completely removing the call to this function, preserving the rest of the changes and see if that works for you too.

Reply:
Scratch that, we need this for adding/removing radios.